iac-extensions/arm/src/test/resources/checks/IpRestrictedAdminAccessCheck/Microsoft.Network_networkSecurityGroups_securityRules/protocol.bicep

```diff
@@ -0,0 +1,57 @@
+// Noncompliant@+1
```

Comment:
Please see relevant JSON and fix code formatting (`apiVersion` is not in new line)